wiki/Article_Style_Guide/en.md

```diff
@@ -63,22 +88,13 @@ For example, use:
   - `zh-tw.md`, or
   - `zh-hk.md`
 
-This is simply to maintain consistency and simplicity while following the ISO 639-1 codes.
+Doing so will allow us to maintain consistency and simplicity while following the ISO 639-1 codes.
```

Comment:
I suggest ``Doing so would allow the other contributors to also maintain consistency and simplicity[..]``